rclpy/src/rclpy/_rclpy.c

```diff
@@ -772,6 +799,8 @@ rclpy_create_timer(PyObject * Py_UNUSED(self), PyObject * args)
 
 /// Returns the period of the timer in nanoseconds
 /**
+ * Raise ValueError if capsule is not a timer capsule
```

Comment:
typo: Raise**s** for consistency with other docblocks added in this PR
Can you also move the `Raise RuntimeError on rcl error` here? (same for the other functions)

Reply:
Fixed 451a072e372eecc46e07096f8fdcb2278051bd54